recipes/slipcover/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "slipcover" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/slipcover-{{ version }}.tar.gz
+    sha256: 80433ab2ccc9d2cda69f3c5adca4d978c34f13efb551c36e32657598b8c207be
+  # missing file from 0.2.0 sdist required for build
+  # fixed in:
+  # https://github.com/plasma-umass/slipcover/pull/31
+  - path: pyptr.h
+  - patches:
+    - 0001-use-env-build-flags.patch
+
+build:
+  skip: true  # [py>=312 or py2k]
```

Comment:
Please omit this skip. It doesn't serve a (good) purpose on conda-forge and will interfere with the Python 3.12 migration once it is going to happen.
```suggestion
```

Reply:
Roger, thanks.

Reply:
I don't exactly know how to solve this. The package doesn't support py3.12 and has compiled components. Should I have version skips or version constraints in the file?